### PR TITLE
(firefoxesr) Migrated to the core team

### PR DESCRIPTION
**Pull request by AdmiringWorm**
There was a request to update this package, so I suggested to move it to the core team packages repo.

https://github.com/chocolatey/chocolatey-package-requests/issues/91

### CODE REVIEW

File: automatic/Firefox/Firefox.nuspec [hunk 1]
```diff
@@ -26,6 +26,7 @@ This package installs Firefox in the first language which matches this list:
 4. If Firefox does not support the system language, it will fall back to “en-US”
 
 Looking for Firefox Developer Edition? Install the [firefox-dev](/packages/firefox-dev) package.
```
Comment (pascalberger) [inline automatic/Firefox/Firefox.nuspec]: Add this in the "Notes" section?

File: automatic/FirefoxESR/FirefoxESR.nuspec [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/firefox.png</iconUrl>
+    <licenseUrl>https://www.mozilla.org/en-US/MPL/2.0/</licenseUrl>
+    <id>FirefoxESR</id>
+    <title>Mozilla Firefox ESR</title>
+    <owners>chocolatey</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+Bringing together all kinds of awesomeness to make browsing better for you.
+
+This package installs Firefox ESR in the first language which matches this list:
```
Comment (pascalberger) [inline automatic/FirefoxESR/FirefoxESR.nuspec]: Add this to notes section?

File: automatic/FirefoxESR/FirefoxESR.nuspec [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <bugTrackerUrl>https://bugzilla.mozilla.org/</bugTrackerUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/firefox.png</iconUrl>
+    <licenseUrl>https://www.mozilla.org/en-US/MPL/2.0/</licenseUrl>
+    <id>FirefoxESR</id>
+    <title>Mozilla Firefox ESR</title>
+    <owners>chocolatey</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+Bringing together all kinds of awesomeness to make browsing better for you.
+
```
Comment (pascalberger) [inline automatic/FirefoxESR/FirefoxESR.nuspec]: Add features from https://www.mozilla.org/en-US/firefox/new/?

```
* **Freedom is fast:** Go anywhere you want on the Web — with a quickness.
* **Freedom is personal:** Enjoy the most built-in privacy tools of any browser.
* **Freedom is yours:** Use the only browser built for people, not profit.
```

File: automatic/FirefoxESR/FirefoxESR.nuspec [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+3. The Windows system language where the Firefox package gets installed.
+4. If Firefox does not support the system language, it will fall back to “en-US”
+
+Looking for Firefox Developer Edition? Install the [firefox-dev](/packages/firefox-dev) package.
+Looking for Firefox for Personal Use? Install the [Firefox](/packages/Firefox) package.
+    </description>
+    <summary>Bringing together all kinds of awesomeness to make browsing better for you</summary>
+    <tags>browser mozilla firefox admin foss cross-platform</tags>
+    <releaseNotes>https://www.mozilla.org/en-US/firefox/45.7.0/releasenotes/</releaseNotes>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/FirefoxESR</packageSourceUrl>
```
Comment (pascalberger) [inline automatic/FirefoxESR/FirefoxESR.nuspec]: Add projectSourceUrl (I think the source is in https://hg.mozilla.org/mozilla-central/)

File: automatic/FirefoxESR/FirefoxESR.nuspec [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0" />
+    </dependencies>
+  </metadata>
+    <files>
+    <file src="tools\**" target="tools" />
+	<file src="..\Firefox\tools\helpers.ps1" target="tools" />
```
Comment (pascalberger) [inline automatic/FirefoxESR/FirefoxESR.nuspec]: Indentation

File: automatic/FirefoxESR/tools/chocolateyInstall.ps1 [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  )
+}
+
+if ($alreadyInstalled -and ($env:ChocolateyForce -ne $true)) {
+  Write-Output $(
+    "Firefox is already installed. " +
+    'No need to download an re-install again.'
+  )
+} else {
+
```
Comment (pascalberger) [inline automatic/FirefoxESR/tools/chocolateyInstall.ps1]: Remove empty line

File: automatic/FirefoxESR/FirefoxESR.nuspec [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+2. If Firefox is already installed: the same language as the already installed Firefox.
+3. The Windows system language where the Firefox package gets installed.
+4. If Firefox does not support the system language, it will fall back to “en-US”
+
+Looking for Firefox Developer Edition? Install the [firefox-dev](/packages/firefox-dev) package.
+Looking for Firefox for Personal Use? Install the [Firefox](/packages/Firefox) package.
+    </description>
+    <summary>Bringing together all kinds of awesomeness to make browsing better for you</summary>
+    <tags>browser mozilla firefox admin foss cross-platform</tags>
+    <releaseNotes>https://www.mozilla.org/en-US/firefox/45.7.0/releasenotes/</releaseNotes>
```
Comment (pascalberger) [inline automatic/FirefoxESR/FirefoxESR.nuspec]: Link needs to be updated in `au_SearchReplace`

File: automatic/FirefoxESR/FirefoxESR.nuspec (hunk 4 above)
Comment (AdmiringWorm) [inline automatic/FirefoxESR/FirefoxESR.nuspec]: projectSourceUrl is available on line 8:
https://github.com/chocolatey/chocolatey-coreteampackages/pull/559/files#diff-699a4871ba1503a6701a96a017af387cR8

**Comment by AdmiringWorm** [comment]
@pascalberger I made the same changes to the firefox package regarding the description and release notes to be the same as FirefoxESR.
Please review them again please!

**Comment by pascalberger** [comment]
@AdmiringWorm LGTM. I'm just not sure if `comm-central` is the correct projectSourceUrl. Based on [this site](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Viewing_and_searching_Mozilla_source_code_online), [mozilla-central](https://hg.mozilla.org/mozilla-central/) is the main repo for recent Firefox and Gecko releases and [comm-central](https://dxr.mozilla.org/comm-central/) for Thunderbird, Seamonkey and a mirror.

**Comment by AdmiringWorm** [comment]
@pascalberger Ah yes, you're right.
I just re-used the project source url from the firefox package, guess I had changed that by an accident when working on the thunderbird package.

**Pull request merged**